src/dal/entities/repository-entity.ts

```diff
@@ -0,0 +1,29 @@
+export interface RepositoryEntity {
+    id: string;
+    deleted: boolean;
+    createdBy?: string;
+    lastUpdatedBy?: string;
+    creationDate: string;
+    lastUpdateDate: string;
+    dataVersion: number;
+    realityId: string;
+    classification?: string;
+    secretGroups?: string[];
+}
+
+export function isRepositoryEntity(object: any): boolean {
+    const joi = require('joi');
```

Comment:
is this the ideal way of checking instance of interface?

Reply:
yes